<launch>
    <node name="theodolite_listener" type="theodolite_listener" pkg="theodolite_node" output="screen">

        <!-- Parameter to set the rate of the listener. Should be between 1 and 100Hz -->
        <param name="rate" value= "5" />        
        <!-- Parameter to show data on terminal if needed. Set True to use it -->
        <param name="show_data" value= "false" />
        <!-- Parameter to set the number of theodolite used. Should be more than 0. -->
        <param name="number_of_theodolite" value= "1" />

        <!-- Parameter to set the number ping for the first synchronization. Should be more than 0. -->
        <param name="number_first_synchronization" value= "50" />
        <!-- Parameter to set the delay between two synchronization for the same theodolite, after the first one during data acquisition. Should be more than 0. Delay in secondes -->
        <param name="delay_synchronization_theodolite" value= "30" />
        <!-- Parameter to set the delay between synchronization of two different theodolites, after the first one during data acquisition. Should be more than 0. Delay in secondes -->
        <param name="delay_synchronization_between_theodolite" value= "5" />
        <!-- Parameter to set the number ping for the others synchronization. Should be more than 0. -->
        <param name="number_others_synchronization" value= "5" />

	</node>
</launch>
